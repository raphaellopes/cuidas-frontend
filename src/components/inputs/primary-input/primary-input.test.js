// vendors
import React from 'react';
import renderer from 'react-test-renderer';

// locals
import PrimaryInput from './index';

const component = props => (renderer.create(
  <PrimaryInput {...props} />,
));

describe('<PrimaryInput />', () => {
  test('Should render the component', () => {
    const tree = component({ children: 'My children' }).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
