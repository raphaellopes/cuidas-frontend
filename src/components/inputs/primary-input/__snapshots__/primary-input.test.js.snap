// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PrimaryInput /> Should render the component 1`] = `
<input
  className="sc-bdVaJa htCbLN"
  color="dark"
  type="text"
>
  My children
</input>
`;
